scripts/unit.sh

```diff
@@ -50,7 +50,7 @@ function unit::run() {
 
   testout=$(mktemp)
   pushd "${BUILDPACKDIR}" > /dev/null
-    if go test ./... -v -run "Unit|Example" | tee "${testout}"; then
+    if go test ./... -v -count=1 -run "Unit|Example" | tee "${testout}"; then
```

Comment:
Note for posterity - this is the idiomatic way to disable test caching in golang.